Code/Framework/AzFramework/Tests/QualitySystemComponentTests.cpp

```diff
@@ -0,0 +1,225 @@
+#include <AzCore/UserSettings/UserSettingsComponent.h>
+#include <AzFramework/Application/Application.h>
+#include <AzFramework/Quality/QualitySystemComponent.h>
+#include <AzFramework/Quality/QualityCVarGroup.h>
+
+namespace UnitTest
+{
+    class QualitySystemComponentTestFixture : public LeakDetectionFixture
+    {
+    public:
+        QualitySystemComponentTestFixture()
+            : LeakDetectionFixture()
+        {
+        }
+
+    protected:
+        void SetUp() override
+        {
+            using FixedValueString = AZ::SettingsRegistryInterface::FixedValueString;
+
+            m_settingsRegistry = AZStd::make_unique<AZ::SettingsRegistryImpl>();
+            AZ::SettingsRegistry::Register(m_settingsRegistry.get());
+
+            m_application = AZStd::make_unique<AzFramework::Application>();
+
+            // setup the runtime paths for the FileTagComponent
+            auto projectPathKey = FixedValueString(AZ::SettingsRegistryMergeUtils::BootstrapSettingsRootKey) + "/project_path";
+            AZ::IO::FixedMaxPath enginePath;
+            m_settingsRegistry->Get(enginePath.Native(), AZ::SettingsRegistryMergeUtils::FilePathKey_EngineRootFolder);
+            m_settingsRegistry->Set(projectPathKey, (enginePath / "AutomatedTesting").Native());
+            AZ::SettingsRegistryMergeUtils::MergeSettingsToRegistry_AddRuntimeFilePaths(*m_settingsRegistry);
+
+            m_console = AZ::Interface<AZ::IConsole>::Get();
+        }
+
+        void TearDown() override
+        {
+            m_application->Stop();
+            m_application.reset();
+
+            AZ::SettingsRegistry::Unregister(m_settingsRegistry.get());
+            m_settingsRegistry.reset();
+        }
+
+        AZ::IConsole* m_console;
+        AZStd::unique_ptr<AzFramework::Application> m_application;
+        AZStd::unique_ptr<AZ::SettingsRegistryInterface> m_settingsRegistry;
+    };
+
+    TEST_F(QualitySystemComponentTestFixture, QualitySystem_Registers_Group_CVars)
+    {
+        auto result = m_settingsRegistry->MergeSettings(R"(
+            {
+                "O3DE": {
+                    "Quality": {
+                        "Groups": {
+                            "q_test": {
+                                "Levels": [ "low", "high" ],
+                                "Default": 1,
+                                "Description": "q_test quality group",
+                                "Settings": {
+                                    "q_test_sub": [0,1]
+                                }
+                            },
+                            "q_test_sub": {
+                                "Levels": [ "low", "high" ],
+                                "Default": 0,
+                                "Description": "q_test_sub quality group",
+                                "Settings": {
+                                    "a_cvar": [123,234]
+                                }
+                            }
```

Comment:
You can leverage this JSON data to test out the enum case for the `Default` field by using the "DefaultQualityLevel" string as well.
```suggestion
                            "q_test_sub": {
                                "Levels": [ "low", "high" ],
                                "Default": "DefaultQualityLevel",
                                "Description": "q_test_sub quality group",
                                "Settings": {
                                    "a_cvar": [123,234]
                                }
                            }
```